src/Http/Middleware/EnsureFrontendRequestsAreStateful.php

```diff
@@ -50,7 +50,7 @@ protected function frontendMiddleware()
             config('sanctum.middleware.encrypt_cookies', \Illuminate\Cookie\Middleware\EncryptCookies::class),
             \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
             \Illuminate\Session\Middleware\StartSession::class,
-            config('sanctum.middleware.validate_csrf_token', config('sanctum.middleware.verify_csrf_token', \Illuminate\Foundation\Http\Middleware\VerifyCsrfToken::class)),
```

Comment:
Isn't this to handle old published configurations?

Reply:
https://github.com/laravel/sanctum/blob/1f79f9d13e79c2af670ce2bb69c0a4ce12f6a8d8/config/sanctum.php#L80

https://github.com/laravel/laravel/blob/7effeb99ec1568f006ed95e60d3e7dfeed1b8160/config/sanctum.php#L80